.github/workflows/node.js.yml

```diff
@@ -51,15 +62,15 @@ jobs:
               with:
                   verbose: true
                   file: ./coverage/coverage-final.json
-                  flags: codewhisperer
+                  flags: windows-unittests
 
-    windows:
-        name: Windows nodejs
-        runs-on: windows-2019
+    qa:
+        name: macOS nodejs QA
+        runs-on: macos-latest
```

Comment:
linux will have a faster filesystem and faster package installs. any reason to macos?